pkg/utils/etcdutil/etcdutil.go

```diff
@@ -532,11 +542,16 @@ func (lw *LoopWatcher) watch(ctx context.Context, revision int64) (nextRevision
 						log.Error("put failed in watch loop", zap.String("name", lw.name),
 							zap.String("key", lw.key), zap.Error(err))
 					}
+					log.Debug("put in watch loop", zap.String("name", lw.name),
+						zap.ByteString("key", event.Kv.Key),
```

Comment:
```suggestion
						zap.String("key", string(event.Kv.Key)),
```

Reply:
Why? What is the difference between them?

Reply:
Oh, they are the same.